packages/react/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+module.exports = () => {
+  const { WITH_REPORT } = process.env;
+
+  return {
+    mode: 'production',
+    entry: './src/index.ts',
+    output: {
+      path: path.resolve(__dirname, 'dist'),
```

Comment:
why do we have 2 different output path? (here `dist` while tsconfig holds `lib`)

Reply:
Removed `Webpack`. Do not support `package` for now and add it when needed.